src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -506,9 +506,9 @@ class BeamSim @Inject() (
       logger.warn(s"Non-negative end times found for person activities - ${activityEndTimesNonNegativeCheck.size}")
     }
 
-    if (beamConfig.beam.outputs.writeGraphs) {
-      // generateRepositioningGraphs(event)
-    }
+//    if (beamConfig.beam.outputs.writeGraphs) {
+//      generateRepositioningGraphs(event)
+//    }
```

Comment:
I assume instead of commenting this out, introducing new configuration value
beamConfig.beam.outputs.writeRepositioningGraphs